From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 18 Jul 2020 16:26:32 +0100
Subject: autoupdate

---
 configure.ac | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index c143af2..44dab36 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
-AC_PREREQ([2.60])
-AC_INIT([simple-mtpfs], [0.3.0], [phatina@gmail.com])
+AC_PREREQ([2.69])
+AC_INIT([simple-mtpfs],[0.3.0],[phatina@gmail.com])
 
 AC_CONFIG_MACRO_DIR([m4])
 
@@ -78,11 +78,12 @@ else
     AC_MSG_WARN([Mounting by real device path is not supported. Fallback to <bus>/<device> notation]);
 fi
 
-AC_OUTPUT([
+AC_CONFIG_FILES([
     makefile
     src/makefile
     man/makefile
 ])
+AC_OUTPUT
 
 AC_MSG_NOTICE([
 
